apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/ServiceProviderIdentifiers.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.bfd.server.war.stu3.providers;
+
+public enum ServiceProviderIdentifiers {
```

Comment:
Public types like this should really have some JavaDoc explaining their purpose, how to use them, etc.